lib/generators/boring/figjam/install/install_generator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Boring
+  module Figjam
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds figjam gem to the app'
+
+      def add_figjam_gem
+        say 'Adding figjam gem', :green
+
+        gem 'figjam'
+      end
+
+      def configure_figjam
+        say 'Configuring figjam', :green
+
+        Bundler.with_unbundled_env do
+          run 'bundle exec figjam install'
+        end
+      end
+    end
```

Comment:
We need to also add application.yml to .gitignore since it will contain sensitive info